stea/__init__.py

```diff
@@ -67,20 +67,31 @@ def calculate(stea_input):
                                  stea_input.project_version,
                                  stea_input.config_date)
     request = SteaRequest(stea_input, project)
-
+  
     for profile_id, profile_data in stea_input.ecl_profiles.iteritems():
-        ecl_key = profile_data[SteaInputKeys.ECL_KEY]
-        start_year = profile_data.get(SteaInputKeys.START_YEAR)
-        end_year = profile_data.get(SteaInputKeys.END_YEAR)
-
-        request.add_ecl_profile(profile_id, ecl_key, first_year=start_year, last_year=end_year)
+        if not profile_id in project.profiles:
+          profile_list=[k for k,v in project.profiles.items() if ('Description' in v and v['Description']==profile_id)]
```

Comment:
You are using a list here - and iterating over a list; but as I can see it you can (and should) only ever get *one* profile out of this?

Reply:
if you give the explicit profile_id then you can only ever get 1 profile. However if you use the description field then multiple profiles can have the same description and be tied to same ecl profile. This happened on real field testing.

Reply:
Well -  the rest of the profile specification is only for one profile.

Reply:
The rest of the profile specification relates to how to filter the eclipse results which are the same for all profile_ids. 

For example in STEA we may have several costs associated with the gas for example. E.g. transport, processing, ... In the STEA cases I saw for testing, it was very common to see this a cost profile = production profile.

Old way to define this:

```yml
ecl-profiles:
  PROFILE_ID_1:
    ecl-key: FGPT
    start-year: ????
  PROFILE_ID_2:
     ecl-key: FGPT
    start-year: ????
  ...
  PROFILE_ID_N:
     ecl-key: FGPT
    start-year: ????
```

This can now be shortened to (assuming PROFILE_DESCRIPTION is set properly in stea project)

```yml
ecl-profiles:
  PROFILE_DESCRIPTION:
    ecl-key: FGPT
    start-year: ????
```